src/Http/Livewire/LivewireDatatable.php

```diff
@@ -1563,4 +1640,39 @@ public function cellClasses($row, $column)
         // Override this method with your own method for adding classes to a cell
         return config('livewire-datatables.default_classes.cell', 'text-sm text-gray-900');
     }
+
+    public function toggleSortDirection(string $direction): string
+    {
+        if (!in_array($direction, self::ORDER_BY_DIRECTION_STATES)) {
+            throw new \Exception("Invalid direction $direction given in toggleSortDirection() method. Allowed values: asc, desc.");
+        }
+
+        switch ($direction) {
+            case $direction === 'desc' :
```

Comment:
I'm used to writing this as `case 'desc':` Any special reason to repeat the `$direction`?
Also, PhpStorm indicates PHP8.0 has a new function `match` that could be used like this:
```
return match ($direction) {
    'desc' => 'asc',
    default => 'desc',
};
```

Reply:
@martintern I am not sure i understand what do you mean with this _I'm used to writing this as case 'desc': Any special reason to repeat the $direction?_. Can you please paste an example?

For the match, i think i cant use it since livewire-datatables is not on php8 yet.